src/ripple/rpc/Context.h

```diff
@@ -60,6 +60,21 @@ struct Context
     Headers headers {};
 };
 
+template <class T>
+struct ContextGeneric
+{
+    beast::Journal j;
+    T params;
+    Application& app;
+    Resource::Charge& loadType;
+    NetworkOPs& netOps;
+    LedgerMaster& ledgerMaster;
+    Resource::Consumer& consumer;
+    Role role;
+    std::shared_ptr<JobQueue::Coro> coro;
+    InfoSub::pointer infoSub;
+};
+
```

Comment:
This duplicates most of the `Context` struct. Maybe have a base class with the common fields? Also `ContextGeneric` isn't a great name. Maybe `GRPCContext` or somesuch.

Reply:
Created a base class and changed the names of the derived classes.